### PR TITLE
add website with styles

### PR DESCRIPTION
**Pull request by M1rn4**
# Add website :100: 
* Add navbar
* Add home page
* Add quote page
* Add calculator page

### CODE REVIEW

File: src/styles/home.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.homePage p {
+  width: 60vw;
+  right: 20vw;
+  text-align: justify;
+  top: 25%;
+}
```
Comment (Danieldotcomcoder) [inline src/styles/home.css]: Kindly try to match the general layout of the wireframes as much as you can: for example in the home page the navbar needs to be in the far right, and the paragraph much bigger and to the left.
image

File: src/styles/calculator.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  margin-left: 10vw;
+  margin-top: 10vh;
+}
+
+.container {
+  width: 80%;
+}
```
Comment (Danieldotcomcoder) [inline src/styles/calculator.css]: kindly try to match the general layout of the wireframes as much as you can, 
image
image
Here the calculator needs to be much smaller and to the right side as in the wireframe

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @M1rn4 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @M1rn4, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**